system-int-test/test/cmr/system_int_test/ingest/bulk_update/bulk_update_test.clj

```diff
@@ -760,6 +762,44 @@
                 :VariableLevel2 "CLOUD LIQUID WATER/ICE"}]
               (:ScienceKeywords (:umm concept)))))))
 
+(deftest bulk-update-update-not-found-test
+  (let [concept-ids (ingest-collection-in-each-format 
+                      find-update-keywords-umm
+                      [:dif])
+        _ (index/wait-until-indexed)
+        bulk-update-body {:concept-ids concept-ids
+                          :name "TEST NAME"
+                          :update-type "FIND_AND_UPDATE"
+                          :update-field "SCIENCE_KEYWORDS"
+                          :find-value {:Topic "DOES NOT EXIST"}
+                          :update-value {:Category "EARTH SCIENCE"
+                                         :Topic "ATMOSPHERE"
+                                         :Term "AIR QUALITY"
+                                         :VariableLevel1 "EMISSIONS"}}
+        task-id (:task-id (ingest/bulk-update-collections "PROV1" bulk-update-body))]
+      (index/wait-until-indexed)
+      (let [collection-response (ingest/bulk-update-task-status "PROV1" task-id)
+            concept-id (first concept-ids)
+            status-message (str "Collection with concept-id [" concept-id
+                                "] is not updated because no find-value found.")]
+        (is (= "COMPLETE" (:task-status collection-response)))
+        (is (= [{:concept-id concept-id,
+                 :status "FAILED",
+                 :status-message status-message}]
+               (:collection-statuses collection-response))))
+
+      ;; Check that each concept was not updated, no new revision-id, format is not changed to umm json.
+      (doseq [concept-id concept-ids
+              :let [concept (-> (search/find-concepts-umm-json :collection
+                                                               {:concept-id concept-id})
+                                :results
+                                :items
+                                first)]]
+       (is (= 1
+              (:revision-id (:meta concept))))
+       (is (not= "application/vnd.nasa.cmr.umm+json"
+              (:format (:meta concept)))))))
+
```

Comment:
I would add one more test that documents your comment in CMR-4608. If performing a find and update with the same value and that value is found in the collection metadata (so the UMM ends up being identical), we perform the update so that there is a new revision created and saved in UMM JSON format.

Reply:
Done.